### DISCUSSION TITLE
Use Ctrl-E instead Ctrl-C to cancel auto-completion

### DISCUSSION THREAD

**Discussion by asumbek** [helix-editor/helix#6220] [Q&A] [answered]
Is there a way to use Ctrl-E instead Ctrl-C to cancel auto-completion? I can't find it the docs

**Comment by the-mikedavis** [comment, marked as answer]
Not currently. There is an issue tracking the ability to remap the key bindings for UI components like completion popups: #5505

**Reply by asumbek** [reply]
Oh, I'll keep track of the issue then. Thank you 😄
